airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -170,9 +170,13 @@ def format_datetime(
     """
     if isinstance(dt, datetime.datetime):
         return dt.strftime(format)
-    dt_datetime = (
-        datetime.datetime.strptime(dt, input_format) if input_format else str_to_datetime(dt)
-    )
+
+    if isinstance(dt, int):
+        dt_datetime = datetime.datetime.fromtimestamp(dt, tz=datetime.timezone.utc)
```

Comment:
Should we distinguish between epoch seconds, milliseconds, microseconds and nanoseconds?

Reply:
input_format should be provided in this case, default "%s"